apps/server/src/modules/files-storage-client/dto/file.dto.ts

```diff
@@ -1,4 +1,5 @@
-import { EntityId, FileRecordParentType } from '@shared/domain';
+import { EntityId } from '@shared/domain';
+import { FileRecordParentType } from '@src/modules/files-storage/entity/filerecord.entity';
```

Comment:
We should switch this parentType. The allowed types in client can be the same like in files-storage core module. But it is also possible that it is only a subset. We shouldt not import it from this place.

Same in the complet client. Different modules, the files-storage module is not importat, also it should not use parts from this place.

Reply:
@SevenWaysDP In the IFileDomainObjectProps we use the parent type from '@shared/infra/rabbitmq' having all the parent types the files storage supports. For the http api we only support task and lesson. Can we reduce the rabbitmq paren type to task and lesson as well?

```
import { EntityId } from '@shared/domain';
import { FileRecordParentType } from '@shared/infra/rabbitmq';

export interface IFileDomainObjectProps {
	id: EntityId;
	name: string;
	parentType: FileRecordParentType;
	parentId: EntityId;
}
```

Reply:
Use rabbitmq FileRecordParentType and rename rabbitmq and files storage FileRecordParentType to FileRecordParent.